### DISCUSSION TITLE
Documents not listed in Dashboard or Documents pages; Empty Statistics section

### DISCUSSION THREAD

**Discussion by shtrom** [jonaswinkler/paperless-ng#1077] [Support] [answered]
Hi,

I've just migrated from the-paperless-project/paperless@ec1d132cb0e6bb8b9eddd60f7f69bb08e93ffcb2 (﻿﻿﻿﻿﻿﻿﻿2.7.0-179-gec1d132) to a docker setup using jonaswinkler/paperless-ng  `latest` de901ac006dd at the moment).

They are using the same `data` and `media` volumes as before, and I can see the documents there. The DB is SQLite.

I used to have `PAPERLESS_DISABLE_LOGIN=true` in the past, which seems to have been using a `consumer` user. I have replaced this with `PAPERLESS_AUTO_LOGIN_USERNAME`, which seems to do what it needs to.

However, I cannot see any of the documents or tags _in the dashboard_.  I can see lists of the documents, tags and correspondents _in the admin section_ (I don't seem to be able to see the scans themselves, but I can see the metadata and OCR'd text).

I have tried creating a `paperless` superuser, following the instructions in the dockerfile, and auto-logging as this user at first. As there weren't any documents in the dashboard, I then tried to auto-login as user `consumer`, which had the same effect (save for no permission to access the Admin UI). I then manually went into the db, deleted the `consumer` user, and changed the id of the `paperless` user to 1, in the hope that all FKs would magically adjust. Same symptoms. Everything works fine, but no document/tags/correspondents in the UI.

Is there a way to change ownership of documents, or even some more-or-less hacky way to fix the DB (e.g., updating/recreating id and fk links)?

**Comment by jonaswinkler** [comment, marked as answer]
If you click on "Documents" in the nav bar, do you see your documents? Is it just the dashboard that does not show any documents? In the "Statistics" section, do you see the count of documents?

**Reply by shtrom** [reply]
Ah, that's what I meant: They show neither in the Dashboard nor the ”Documents” section.

Looks like I have a non-existent number of documents
![Screen Shot 2021-05-31 at 10 09 04](https://user-images.githubusercontent.com/160280/120124559-3b8bcd00-c1f8-11eb-9c1d-2f7066772d81.png)

**Reply by jonaswinkler** [reply]
This looks like a communication error. Could you press F12, reload, and check if there are any failed network requests?

**Reply by shtrom** [reply]
Ah yep! It seems some websockets and XHR aren't going through (sent, but no reply or error; I expect they will timeout). I'm proxying via NGINX.

**Reply by shtrom** [reply]
Ok, I'll try this out and report https://paperless-ng.readthedocs.io/en/latest/setup.html?highlight=nginx#using-nginx-as-a-reverse-proxy

**Reply by shtrom** [reply]
Ok, that fixed the WebSocket error, but I now see permission issues in the logs

```
webserver_1  | [2021-05-31 00:31:09,669] [WARNING] [django.request] Unauthorized: /paperless/api/saved_views/
webserver_1  | [2021-05-31 00:31:10,210] [WARNING] [django.request] Unauthorized: /paperless/api/statistics/
```

**Reply by shtrom** [reply]
Ok, fixed, that was some weirdness with my site-wide HTTP Auth. I created a matching user and that worked.

Looking at the network tab definitely helped flag what was wrong.

So, tl;dr:
* Needed to enable websocket support https://paperless-ng.readthedocs.io/en/latest/setup.html?highlight=nginx#using-nginx-as-a-reverse-proxy
* Needed to have a Paperless user matching the HTTP Auth allowed user, so XHRs are allowed.

Thanks for your help @jonaswinkler!
